include/in_process_server.h

```diff
@@ -206,6 +207,7 @@ class Client
     wl_shm* shm() const;
     wl_data_device_manager* data_device_manager() const;
     wl_seat* seat() const;
+    struct zwp_primary_selection_device_manager_v1* primary_selection_device_manager() const;
```

Comment:
Non-blocking but you don't need the `struct` keyword, and I'm trying to move away from using it in this case.